src/cliClient.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const inquirer = require('inquirer');
+
+function buildWorkspaceNames(workspaces) {
+    let str = '';
+    let i = 1;
+    workspaces.forEach(element => {
```

Comment:
I think we should use `reduce` instead.

What you think?

Reply:
Done.